docs/resources/environments_keytab.md

```diff
@@ -0,0 +1,54 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cdp_environments_keytab Resource - terraform-provider-cdp"
+subcategory: ""
+description: |-
+  
+---
+
+# cdp_environments_keytab (Resource)
```

Comment:
The docs probably need to be regenerated

Reply:
updated